walkthroughs/howto-cross-account/README.md

```diff
@@ -0,0 +1,128 @@
+
+* The first backend is dockercloud/hello-world 
+* The secondary backend is karthequian/helloworld
+
+These backends will be configured in distinct accounts and made accessible through the gateway. We will manipulate routes that show that we are able to migrate traffic from one destination to another using the App Mesh APIs.
+
+## Setup
+
+1. Clone this repository and navigate to the walkthroughs/howto-cross-account folder, all commands will be ran from this location.
+    ```
+    cd walkthroughs/howto-cross-account
+    ```
+2. Edit the `vars.env` file to add your account and profile settings:
+    
+    1. **Project Name** used to isolate resources created in this demo from other's in your account. e.g. howto-cross-account
+        ```
+        export PROJECT_NAME=howto-cross-account
+        ```
+    2. **Account IDs**:
+        This demo needs two accounts to which are in the same AWS Organization.
+        ```
+        export AWS_PRIMARY_ACCOUNT_ID=<your_first_account_id>
+        export AWS_SECONDARY_ACCOUNT_ID=<your_secondary_account_id>
+        ```
+        The first account need not be the primary account in the AWS Organization. If it is not the primary account, 
+        please make sure to run the following command from the primary account:
+        `aws ram enable-sharing-with-aws-organization` 
+    3. **AWS Profiles**:
+        Unset the following env variables so that the credentials are not overridden:
+        ```
+        unset AWS_ACCESS_KEY_ID
+        unset AWS_SECRET_ACCESS_KEY
+        ```
+        Set the profile name for each of the accounts mentioned above, each profile should have credentials set:
+        ```
+        export AWS_PRIMARY_PROFILE=...
+        export AWS_SECONDARY_PROFILE=...
+        ```
+        Eg.:
+        ```
+        cat ~/.aws/credentials
+        [default]
+        aws_secret_access_key = ...
+        aws_access_key_id =  ...
+        [primary]
+        aws_secret_access_key = ...
+        aws_access_key_id =  ...
+        [secondary]
+        aws_access_key_id = ...
+        aws_secret_access_key = ...
+        ```
+    4. **Region** e.g. us-west-2
+        ```
+        export AWS_DEFAULT_REGION=us-west-2
+        ```
+    5. **ENVOY_IMAGE** set to the location of the App Mesh Envoy container image, see https://docs.aws.amazon.com/app-mesh/latest/userguide/envoy.html
+        ```
+        export ENVOY_IMAGE=840364872350.dkr.ecr.${AWS_DEFAULT_REGION}.amazonaws.com/aws-appmesh-envoy:v1.12.2.1-prod
+        ```
+    6. **Backend Images**
+        ```
+        export BACKEND_1_IMAGE=dockercloud/hello-world
+        export BACKEND_2_IMAGE=karthequian/helloworld 
+        ```     
+    7. **KEY_PAIR** set to the name of an EC2 key pair. We will use this key pair to access a bastion host in the generated VPC to look at the stats collected by the Envoy proxy. See https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html
```

Comment:
We don't seem to use the bastion..

Reply:
I'd just remove it.

Reply:
Bastion host is really helpful for debugging. I will update the Readme.